regions/io/crtf/test/test_crtf_language.py

```diff
@@ -0,0 +1,128 @@
+import os
+import distutils.version as vers
+
+from numpy.testing import assert_allclose
```

Comment:
`assert_allclose` is apparently unused. Could you check the imports in the test files?

Reply:
sorry , I will remove the unused imports.